### PR TITLE
Turtlelord26 Melee 2.1 Maneuvers

### PR DESCRIPTION
**Pull request by Turtlelord26**
Re #365 and #366 

New Grappling, Shoving, Kicking, Tripping, and Disarming rules.
Blunt damage rule moved to a more logical location and edited to use a less arbitrary number.

Oh, and the first commit below is a pile of language fixes enumerated in the commit note.

### CODE REVIEW

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -788,26 +799,90 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
...
-
-Kicking/Shoving:
-	Instead of dealing full damage, a character can try to kick/shove a human sized
-opponent to force movement by rolling for a normal unarmed/armed melee attack 
-with an additional -2 miss penalty against the opponent's Guard DC. If the attack 
-succeeds, it yields 1/2 the damage it normally would and pushes the target up to
-1m for every 5 strength you have.
+	Grappling is a maneuver where a character physically grabs another, 
+attempting to immobilize the target. To initiate a grapple, a character initiates 
+an opposed maneuver check. If the target wins, the grapple fails and nothign happens.
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: nothign

File: 01_01_Basic_Rules.txt [hunk 2]
```diff
@@ -788,26 +799,90 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
...
+If the grappler wins, both characters gain the Grappled state, and the attacker 
+starts with control.
+
+	While in the Grappled state, ranged attacks at range R2 or R3 (based on 
+the weapon used) against either character have a 50% chance to hit the other. Also, 
+the characters are limited in the actions they can take, can only target each 
+other, and may use only one Action per round. The character with control of the 
+grapple may make Unarmed Strikes, use other combat maneuvers, attempt to Pin, 
+or disengage. The character without control may make Unarmed Strikes, attempt to 
+disengage, or attempt to gain control. Each efforts takes one Action.
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: efforts

**Review by rosebrallier — APPROVED** [review]
Lovely; I tagged a couple of typos

File: 01_01_Basic_Rules.txt [hunk 3]
```diff
@@ -736,44 +736,55 @@ push you below the minimum miss chance.
...
-within reach of one of the combatant's melee weapons. At this point (unless one 
-character has a CQB weapon) both characters are assumed to be using their fists 
-or other melee weapons. If a character wishes to leave melee combat, they incur 
-an "Opportunity Attack."  
+	A character of standard size occupies 1m of space. This is not the physical 
+space occupied, but rather both the area effectively controlled by the character and 
+the minimum necessary to effectively fight.
+
+	Two characters engaged in melee combat are considered to be in adjacent 1m 
+spaces, not in the same 1m space. Fights don't gemerally start face-to-face unless
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: gemerally -> generally

File: 01_01_Basic_Rules.txt [hunk 4]
```diff
@@ -736,44 +736,55 @@ push you below the minimum miss chance.
...
+	Two characters engaged in melee combat are considered to be in adjacent 1m 
+spaces, not in the same 1m space. Fights don't gemerally start face-to-face unless 
+you're at a bar.
+
+	Once a character has attempted to attack another at melee range (1m), 
+they are both considered to be in melee combat. They can make attacks against 
+each other with unarmed strikes, melee weapons (improvised or not), or CQB guns 
+as normal. Attacking a target more than 1m away, using a non-CQB weapon, or 
+moving away from the melee (under the character's own power: explosion knockback 
+does not count) will incur an Opportunity Attack as described below.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt:752]: Easy to read and remember! Great improvement!

File: 01_01_Basic_Rules.txt [hunk 5]
```diff
@@ -736,44 +736,55 @@ push you below the minimum miss chance.
...
-walks adjacent (within melee range) to a character that is aware of their 
-opponent. Characters also incur Opportunity Attacks by performing a non-combat 
-skill check when locked in melee combat (such as trying to repair their own 
-armor) or by trying to shoot a different opponent with a non-CQB weapon.
+	A free attack* that is triggered when an opponent leaves melee combat, 
+walks past (within melee range) an alert character, performing a skill check 
+when engaged in melee combat that requires the use of the character's primary 
+manipulator limbs (So a medicine check to stabilize an ally would incur an 
+attack, but an acrobatics check to kick away a grenade would not) or by 
+trying to shoot an opponent more than 1m away while engaged in melee combat.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: Great examples! Again, brief enough to keep me interested as I read, long enough and specific enough for me to understand and make a ruling! Fantastic!

File: 01_01_Basic_Rules.txt [hunk 6]
```diff
@@ -736,44 +736,55 @@ push you below the minimum miss chance.
...
 
-*The free attack can be made with a CQB weapon or a melee weapon (depending on 
-if the weapon is already equipped in the character's hands). If the free attack 
-is made with a CQB weapon, and the weapon is not already aimed at the target, 
-standard Hip Firing rules apply.
+	The free attack can be made with a drawn CQB weapon or a melee weapon. 
+If the free attack is made with a CQB weapon, and the weapon is not already 
+aimed at the target, standard Hip Firing rules apply.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: Should this be a footnote, or another paragraph? Great addendum regardless!

File: 01_01_Basic_Rules.txt [hunk 7]
```diff
@@ -736,44 +736,55 @@ push you below the minimum miss chance.
...
 
 Making Melee Attacks:
-	Melee combat is relatively simple. To hit an enemy player: add your 
-dexterity combat modifier to a d10 roll plus your combat modifier from your 
-weapon/attack*. The result must be greater than the opponent's Guard DC: 
+	Melee accuracy checks use a d10 roll, modified by your dexterity 
+combat modifier and your weapon's accuracy modifier. The result must be greater 
+than the opponent's Guard roll.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: "must be greater than the opponent's Guard Roll"... suggest: ..." in order to hit. If the roll is less than or equal to, the melee attack misses."

File: 01_01_Basic_Rules.txt [hunk 8]
```diff
@@ -736,44 +736,55 @@ push you below the minimum miss chance.
...
+	Defending players roll a Guard check with a d10, modified by the 
+character's Guard stat, which is calculated by [---]. If you successfully
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: Calculated by [---] ??? Is this going to be merged in a separate commit?

File: 01_01_Basic_Rules.txt [hunk 9]
```diff
@@ -788,26 +799,96 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
...
 
-Hitting What You Intend:
-	If you roll 3+ above your miss chance (3+ above the opponent's guard DC), 
-you ignore the opponent's armor and deal full damage. 
+== Combat Maneuvers ==
+
+	In addition to smashing an opponent directly, there are other offensive 
+actions that characters in melee combat may take. Also, some exotic weapons may 
+allow these maneuvers to be performed even outside melee: individual weapon 
+rules take precedence.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt:809]: Great intro!

File: 01_01_Basic_Rules.txt [hunk 10]
```diff
@@ -788,26 +799,96 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
...
-succeeds, it yields 1/2 the damage it normally would and pushes the target up to
-1m for every 5 strength you have.
+	Grappling is a maneuver where a character physically grabs another, 
+attempting to immobilize the target. To initiate a grapple, a character initiates 
+an opposed maneuver check. If the target wins, the grapple fails and nothing happens. 
+If the grappler wins, both characters gain the Grappled state, and the attacker 
+starts with control.
+
+	While in the Grappled state, ranged attacks at range R2 or R3 (based on 
+the weapon used) against either character have a 50% chance to hit the other. Also,
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: As a house rule, I will always add the shooter's luck mod to hit the character they intend. You may not want the complexity creep in the core rules. Up to you.

File: 01_01_Basic_Rules.txt [hunk 11]
```diff
@@ -788,26 +799,96 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
...
+	While in the Pinned state, a character may only attempt to disengage, and 
+success will return the character to the grappled state, without control, instead 
+of releasing the character from the grapple. The character grappling with a Pinned 
+target may attempt to restrain the target with a further opposed maneuver check and 
+appropriate tool, such as zip cuffs. Success renders the target helpless (until 
+they can break the bonds, which depends on the item used, see item descriptions). 
+Pinned targets are also vulnerable to knockout and kill actions: both render the 
+target unconscious and reduce him or her to 1 health, but a successful kill action 
+also forces the target to make a death saving throw. Knockout and Kill require two 
+consecutively won opposed maneuver checks, andtargets additionally add their Shock
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: andtargets -> and targets

File: 01_01_Basic_Rules.txt [hunk 12]
```diff
@@ -788,26 +799,96 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
...
+target may attempt to restrain the target with a further opposed maneuver check and 
+appropriate tool, such as zip cuffs. Success renders the target helpless (until 
+they can break the bonds, which depends on the item used, see item descriptions). 
+Pinned targets are also vulnerable to knockout and kill actions: both render the 
+target unconscious and reduce him or her to 1 health, but a successful kill action 
+also forces the target to make a death saving throw. Knockout and Kill require two 
+consecutively won opposed maneuver checks, andtargets additionally add their Shock 
+modifier to their roll when defending against attempts to kill. If a character 
+restrians, knocks out, or kills a grappled target, he or she is still grappled and 
+must spend an Action to disengage.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: This is so brilliant! Did you get this from DnD or come up with this yourself?

**Review by bleehu — CHANGES_REQUESTED** [review]
Some minor typos. This is super cool! I'll have to re-write some of my enemies to use these systems. Great work!

File: 01_01_Basic_Rules.txt (hunk 6 above)
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt]: As it is vital rules text, I prefer paragraph.

File: 01_01_Basic_Rules.txt (hunk 8 above)
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt]: Yep, that's a dev note. I promise to purge it in a future PR before the superbranch goes to dev.

File: 01_01_Basic_Rules.txt (hunk 12 above)
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt]: I wasn't deliberately using any reference material, but I suspect it was partly inspired by 3.5 grappling mechanics.

File: 01_01_Basic_Rules.txt [hunk 13]
```diff
@@ -736,44 +736,57 @@ push you below the minimum miss chance.
...
+	Opportunity attacks can be made with a drawn CQB weapon or a melee weapon. 
+If the free attack is made with a CQB weapon and the weapon is not already 
+aimed at the target, standard Hip Firing rules apply.
 
 Making Melee Attacks:
-	Melee combat is relatively simple. To hit an enemy player: add your 
-dexterity combat modifier to a d10 roll plus your combat modifier from your 
-weapon/attack*. The result must be greater than the opponent's Guard DC: 
+	Melee attacks make accuracy rolls with a d10, increased by your 
+dexterity combat modifier and your weapon's accuracy modifier. To successfully
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt:768]: I think I forgot to mention in my last review how much I like making one D10 the standard for attack/defense rolls. Great subtle change!

File: 01_01_Basic_Rules.txt [hunk 14]
```diff
@@ -788,26 +801,107 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
...
+	Grappling-specific actions are described below.
+	
+Pin: 	The attacker and defender make an opposed maneuver check. If the attacker 
+	wins, the defender gains the Pinned state.
+Disengage: Characters in control of a grapple may disengage without rolling. 
+	Otherwise both characters choose one of STR-Athletics or DEX-Acrobatics 
+	and roll an opposed check (choices may differ). If the disengager wins, 
+	the grapple ends (and the active character has 3 Actions available this 
+	round).
+Gain Control: The involved characters make an opposed maneuver check as if the
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt:873]: If these are supposed to be topics with paragraphs, I believe (from looking at the commit view) that a newline will need to be inserted after the colon.

**Review by bleehu — APPROVED** [review]

File: 01_01_Basic_Rules.txt (hunk 14 above)
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt:873]: I was hoping those would be read as a definition list. The individual maneuvers are already at topic level so I can't put subsections of grappling on the topic level and there isn't a level underneath it.

In retrospect, I should have put all of the Actions in that list or in the preceding prose instead of dividing them between. I'll log that and the possible style problem as an Issue to keep an eye on (from a clarity perspective).
